tests/test_fixlengths.rs

```diff
@@ -27,6 +35,43 @@ fn prop_fixlengths_all_maxlen() {
 }
 
 #[test]
+fn prop_fixlengths_all_maxlen_trims() {
```

Comment:
Could you drop the `prop_` prefix on these test names? (These tests aren't using quickcheck.)